MC/bin/o2dpg_sim_workflow.py

```diff
@@ -730,6 +738,11 @@ def createRestDigiTask(name, det='ALLSMALLER'):
   vertexQCtask['cmd'] = 'o2-qc --config json://${O2DPG_ROOT}/MC/config/QC/json/vertexing-qc-direct-mc.json --remote-batch vertexQC.root ' + getDPL_global_options()
   workflow['stages'].append(vertexQCtask)
 
+  TOFMatchQCneeds = ['TOFMatchQC_local'+str(tf) for tf in range(1, NTIMEFRAMES + 1)]
+  TOFMatchQCtask = createTask(name='TOFMatchQC_finalize', needs=TOFMatchQCneeds, cwd=qcdir, lab=["QC"], cpu=1, mem='2000')
```

Comment:
not sure here. This looks like you are using the same variable name `TOFMatchQCtask` twice

Reply:
I will check tomorrow, but I think I did as Piotr did for the vertex, isn't it? Sorry if I overlooked something.

Reply:
Ciao, 
@knopers8 can confirm, but the vertexQC does the same. So I guess it is fine? Both are added 2x.
Chiara

Reply:
They are in different scopes: one in the for loop for TFs, another one at the end. I suppose this should be fine, unless I am wrongly assuming that variables in Python are destroyed after quitting a for loop?

Reply:
Yes, indeed, I also saw that. But I am not a python expert.

Reply:
Running it over one TF uses basically the same mechanism as when running over multiple TFs. However, the objects are not merged, because there is no need. That being said, if other tasks work fine, I would expect it to fail only if objects in TOF matching task are not mergeable.

Reply:
Yes, I fully agree. If it works for the vertexQC (or other tasks), it will work for TOF. 

This means that we can merge the PR?